### PR TITLE
Crc

### PR DESCRIPTION
**Pull request by bkerler**
This is the change requested by commit #309.

### CODE REVIEW

**Comment by whitequark** [comment]
Just to clarify, how have you tested this?

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Debugging new applets can be hard, especially if bidirectional buses are involve
 You will need git and Python 3.7 (or a newer version, in which case replace `3.7` with that version below). On a Debian or Ubuntu system these can be installed with:
 
     apt-get install --no-install-recommends git python3.7 python3-setuptools \
-      python3-libusb1 python3-aiohttp python3-bitarray python3-crcmod
+      python3-libusb1 python3-aiohttp python3-bitarray
```
Comment (attie) [inline README.md:88]: add `crc` (like in pyproject.toml)

**Review by attie — CHANGES_REQUESTED** [review]
Please also:

- drop the "Remove led example" commit
- merge remaining commits
- discuss testing / validation steps

File: README.md (hunk 1 above)
Comment (whitequark) [reply README.md:88]: No need; we only suggest installation of packages with native components here, and `crc` doesn't have any.

**Comment by bkerler** [comment]
> Just to clarify, how have you tested this?

I've conpared the crc output of each function to be the same using a test input.

**Comment by bkerler** [comment]
> Please also:
> 
>     * drop the "Remove led example" commit
That would require to completely drop this pr and reissue it. Can do that of course if really required.

> 
>     * merge remaining commits
See above.

> 
>     * discuss testing / validation steps

Actually I can add selftest functionality for test automation but that was out of the scope as I didn't see any test automation being used in this project before.

File: software/glasgow/applet/sensor/scd30/__init__.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def __init__(self, interface, logger):
     def _log(self, message, *args):
         self._logger.log(self._level, "SCD30: " + message, *args)
 
-    _crc = staticmethod(crcmod.mkCrcFun(0x131, initCrc=0xff, rev=False))
+    _crc = staticmethod(Calculator(Configuration(width=8,polynomial=0x131,init_value=0xff)).checksum)
```
Comment (whitequark) [inline software/glasgow/applet/sensor/scd30/__init__.py:50]: So this doesn't seem like an equivalent change:
```
>>> import crcmod,crc
>>> crcmod.mkCrcFun(0x11021, initCrc=0xffff, rev=False)(b"abcd")
11510
>>> crc.Calculator(crc.Configuration(width=8,polynomial=0x131,init_value=0xff)).checksum(b"abcd")
220
```

**Review by whitequark — CHANGES_REQUESTED** [review]

**Review by whitequark — CHANGES_REQUESTED** [review]
(Ignore the earlier comment; it was a copy-paste error.)

Could you replace `from crc import Calculator, Configuration` with `import crc` everywhere, please? The `Calculator` and `Configuration` names are pretty generic and I could see them clashing with some other names inside applets potentially.

Looks good to me otherwise.

**Comment by bkerler** [comment]
> 

Yes of course. I will close this pr and will reissue a new combined pr with your requested change.

**Comment by whitequark** [comment]
FYI, you can force-push the PR branch to update the PR with completely new code.

**Pull request closed without merging**